stubs/PyScreeze/pyscreeze/__init__.pyi

```diff
@@ -142,6 +142,8 @@ def locateCenterOnScreen(
     step: int = 1,
     confidence: None = None,
 ) -> Point | None: ...
+def locateOnScreenNear(image: str | Image.Image | _Mat, x: int, y: int) -> Point: ...
```

Comment:
From my reading of the code, this should be:

```suggestion
def locateOnScreenNear(image: str | Image.Image | _Mat, x: int, y: int) -> Box: ...
```

Reply:
You're right, thanks